From 572fd569bff28f25e83b8466c77ccb821675771a Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Sun, 13 Nov 2022 13:36:24 +0100
Subject: [PATCH 4/5] Revert "arch/arm/Kconfig: prepare for rt patch"

This reverts commit 8e60810e199661b607557e560b329097dc3fea0e.
Revert the changes made to make the RT-Patch apply.

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 arch/arm/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 0e8631b96e0f..d5b9364334e0 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -128,6 +128,7 @@ config ARM
 	select PERF_USE_VMALLOC
 	select HAVE_POSIX_CPU_TIMERS_TASK_WORK if !KVM
 	select RTC_LIB
+	select SET_FS
 	select SYS_SUPPORTS_APM_EMULATION
 	select TRACE_IRQFLAGS_SUPPORT if !CPU_V7M
 	# Above selects are sorted alphabetically; please add new ones
-- 
2.35.3

